src/EnergyPlus/HeatingCoils.cc

```diff
@@ -1514,6 +1514,9 @@ namespace HeatingCoils {
 			LoopHeatingCoilMaxRTF = max( LoopHeatingCoilMaxRTF, HeatingCoilLoad / HeatingCoil( CoilNum ).NominalCapacity );
 		}
 
+		// set outlet node temp so parent objects can call calc directly without have to simulate entire model
+		Node( HeatingCoil( CoilNum ).AirOutletNodeNum ).Temp = HeatingCoil( CoilNum ).OutletAirTemp;
+
```

Comment:
And I added Node.Temp at end of each calc routine so UnitarySystem could call Calc during iterations instead of the entire Sim routine (i.e., including UpdateHeatingCoil to set node data).